templates/android.js

```diff
@@ -29,12 +30,87 @@ android {
 }
 
 repositories {
+    maven {
```

Comment:
This changeset is also in #51, right? 🙂

Reply:
Yes, it should be gone after #51 or #39 is merged and I rebase. It's necessary in order for the project to build.